docs/topics/testing/tools.txt

```diff
@@ -1683,15 +1683,17 @@ your test suite.
 
     Output in case of error can be customized with the ``msg`` argument.
 
-.. method:: TransactionTestCase.assertQuerysetEqual(qs, values, transform=repr, ordered=True, msg=None)
+.. method:: TransactionTestCase.assertQuerysetEqual(qs, values, transform=None, ordered=True, msg=None)
 
-    Asserts that a queryset ``qs`` returns a particular list of values ``values``.
+    Asserts that a queryset ``qs`` matches a particular iterable of values
+    ``values``.
 
-    The comparison of the contents of ``qs`` and ``values`` is performed by
-    applying ``transform`` to ``qs``. By default, this means that the
-    ``repr()`` of each value in ``qs`` is compared to the ``values``. Any other
-    callable can be used if ``repr()`` doesn't provide a unique or helpful
-    comparison.
+    If ``transform`` is provided, ``values`` is compared to a list produced by
+    applying ``transform`` to each member of ``qs``.
+
+    If ``transform`` is not provided and ``values`` is a list of strings, it's
+    compared to a list produced by applying ``repr()`` to each member of
+    ``qs``.
```

Comment:
This behavior will be removed in Django 4.1. 

I know we don't say that normally... — but can we somehow highlight "Don't use this!" 

Maybe the description of this behaviour can be moved to the `.. deprecated::` block. (Someone beginning to use this method shouldn't use it right? 🤔)

Reply:
Done.